### PR TITLE
add eventbus for kafka

### PR DESCRIPTION
**Pull request by jkralik**
# Summary
Add event bus for Kafka

# How to test
Use steps 1 and 2 from: https://kafka.apache.org/quickstart

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -43,3 +44,16 @@ services:
       - pubsub
       - start
       - "--host-port=0.0.0.0:8793"
+
+  kafka:
+    image: wurstmeister/kafka
```
Comment (ondrejtomcik) [inline docker-compose.yml]: Use the "more official" image from Kafka. I would propose to use https://hub.docker.com/r/confluent/kafka/ and also stick image version it should supposed to work. Test may fail in the future when new version with breaking change is introduced. Same with zookeeper.

**Comment by giautm** [comment]
@jkralik : Hi, thank you for your implement, but I think it should be live in a separate repository like `eh-kafka` (Issues: https://github.com/looplab/eventhorizon/issues/147). Then, we can add more feature to that repos, like https://github.com/looplab/eventhorizon/issues/163

**Comment by maxekman** [comment]
Cool stuff!

Sounds good to publish as a separate repo. I have not been able to move the other implementations out yet. Be sure to still run the acceptance tests from here. Also make a PR here with a link in the readme.

**Comment by jkralik** [comment]
I moved event bus kafka to: https://github.com/Kistler-Group/eh-kafka

For Kafka i have problem with timeout because initialization of subscription is too slow. With this patch you can define timeout per test.

File: eventbus/acceptance_testing.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ import (
 //       eventbus.AcceptanceTest(t, bus1, bus2)
 //   }
 //
-func AcceptanceTest(t *testing.T, bus1, bus2 eh.EventBus) {
+func AcceptanceTest(t *testing.T, timeout time.Duration, bus1, bus2 eh.EventBus) {
```
Comment (maxekman) [inline eventbus/acceptance_testing.go]: Put timeout as last arg.

Comment (jkralik) [reply eventbus/acceptance_testing.go]: Done! + added link to kafka

**Comment by jkralik** [comment]
@maxekman This fails in test that I didn't touch. Could pls check why?

**Comment by maxekman** [comment]
Great! Yeah, there is unfortunately a test that is unstable.

**Pull request merged**